frontends/web/src/app.tsx

```diff
@@ -203,7 +203,7 @@ export const App = () => {
                 return null;
               })
             }
-            <div className={styles.contentContainer}>
+            <div id="content-container" className={styles.contentContainer}>
```

Comment:
Nit: In the future we should look into a different solution that doesn't need  an id and can be reused, maybe in a component.